static/beta/prod/navigation/application-services-navigation.json

```diff
@@ -57,14 +57,14 @@
       "groupId": "trusted-content",
       "title": "Source Code Security",
       "icon": "shield",
-      "navItems": [
+      "routes":[
```

Comment:
You need to add `"expandable": true` if you want expandable navigation

Reply:
We do not want it to be expandable, only to have 2 entries in the `Source code security` menu, one for overview and one for trusted-artifact-signer.